docs/source/markdown/podman-create.1.md

```diff
@@ -840,21 +840,27 @@ Specify the policy to select the seccomp profile. If set to *image*, Podman will
 
 Note that this feature is experimental and may change in the future.
 
-#### **\-\-secret**=*secret*
+#### **\-\-secret**=*secret*[**,**opt=opt]
```

Comment:
I'm not sure what you're trying to show here, but it comes out as the below, but all bolded:
```
--secret=secret[**,**opt=opt]
```